### PR TITLE
Update video source to advanced bipbop

### PR DESCRIPTION
**Pull request by forbesjo**
## Description
Update the page source to use advanced bipbop, for some reason the previous source does not play in Safari.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -87,7 +87,7 @@
           -->
         <video id=example-video width=960 height=400 class="video-js vjs-default-skin" controls>
           <source
-             src="//d1sh0fycs31s5a.cloudfront.net/eleph/index.m3u8"
+             src="https://s3.amazonaws.com/_bc_dml/example-content/bipbop-advanced/bipbop_16x9_variant.m3u8"
```
Comment (gkatsev) [inline index.html]: you should use the cloudfront url for this.
`//d2zihajmogu5jn.cloudfront.net/bipbop-advanced/bipbop_16x9_variant.m3u8`

**Comment by gkatsev** [comment]
LGTM

**Comment by BrandonOCasey** [comment]
LGTM

**Comment by BrandonOCasey** [comment]
also does this fix #816?

**Comment by forbesjo** [comment]
Yes it does fix that issue

**Pull request merged**